control/tests/statesp_test.py

```diff
@@ -341,18 +341,14 @@ def test_evalfr(self, dt, omega, resp):
         else:
             s = omega * 1j
 
-        # Correct version of the call
+        # Correct versions of the call
         np.testing.assert_allclose(evalfr(sys, s), resp, atol=1e-3)
-        # Deprecated version of the call (should generate warning)
-        with pytest.deprecated_call():
+        np.testing.assert_allclose(sys(s), resp, atol=1e-3)
+
+        # Deprecated version of the call (should generate error)
+        with pytest.raises(AttributeError):
             np.testing.assert_allclose(sys.evalfr(omega), resp, atol=1e-3)
 
-        # call above nyquist frequency
-        if dt:
-            with pytest.warns(UserWarning):
-                np.testing.assert_allclose(sys._evalfr(omega + 2 * np.pi / dt),
-                                           resp,
-                                           atol=1e-3)
```

Comment:
?

Reply:
cant figure out specific question here (maybe due to unfamiliartuy with pytest). 
* makes sense to me that calling sys.evalfr should raise attribute error because it has been removed (must use evalfr(sys, s) now)
* above nyquist test moved elsewhere?

Reply:
Okay it is tested in freqresp_test.py::test_discrete